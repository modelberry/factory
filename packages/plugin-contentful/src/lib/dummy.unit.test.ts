describe('dummy should', () => {
  test('run well', async () => {
    expect({}).toMatchSnapshot()
  })
})
