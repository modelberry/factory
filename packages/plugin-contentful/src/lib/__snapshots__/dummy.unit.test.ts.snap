// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dummy should run well 1`] = `Object {}`;
